openwisp_monitoring/device/api/views.py

```diff
@@ -200,6 +201,22 @@ def _write(self, request, pk):
                 metric.write(client['mac'])
             if created:
                 self._create_clients_graph(metric)
+        if 'resources' in data:
+            resources = {}
+            if 'load' in data['resources']:
+                resources['load'] = 100 * data['resources']['load'][1]
```

Comment:
Why multiplying by 100? This would be confusing for the enduser and Influxdb allows float values.

Reply:
Threshold value can only be an integer. From what I know, load is a value less than 1 (in case it's not the case please correct me), hence had to do it

Reply:
If the reason of multiplying the value * 100 is that Threshold only allows integers then I think it makes more sense to change Threshold.

For the moment we can just create the Metric and skip the Threshold for this case.